File: Demo-LLP-MIP/Demo-LLP-MIP-PHP/demo/IsignCardBinDemo.php
<?php

require_once('D:\file\lianlian\code\Demo-LLP-MIP-PHP\client\LLianPayClient.php');
require_once 'D:\file\lianlian\code\Demo-LLP-MIP-PHP\config\LLianPayConstant.php';
require_once('D:\file\lianlian\code\Demo-LLP-MIP-PHP\utils\LLianPayDateUtils.php');
require_once('D:\file\lianlian\code\Demo-LLP-MIP-PHP\v1\isign\argument\cardbin\CardBinParams.php');
require_once('D:\file\lianlian\code\Demo-LLP-MIP-PHP\v1\info\CardInfo.php');

$params = new CardBinParams();
$params->mch_id = $MchId;
$params->card_no = "3568855594328587";

$url = "https://test.lianlianpay-inc.com/query/v1/isign/cardbin";

$params_json = $params->to_dict();
$result = (new LLianPayClient)->sendRequest($url, $params_json);
echo "\n响应结果：" . print_r($result, true);

File: Demo-LLP-MIP/Demo-LLP-MIP-Python/demo/IsignCardBinDemo.py
import logging
log = logging.getLogger('IsignCardBinDemo')
log.setLevel(logging.INFO)
import json
from client.LLianPayClient import LLianPayClient
from config.LLianPayConstant import MchId
from v1.isign.argument.cardbin.CardBinParams import CardBinParams

params=CardBinParams()
params.mch_id = MchId
params.card_no="3568855594328587"

url = "https://test.lianlianpay-inc.com/query/v1/isign/cardbin"

params_dict = params.to_dict()
params_json = json.dumps(params_dict)
params_json = params_json.replace(" ", "")
result = LLianPayClient().send_request(url, params_json)
log.info(result)